<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SampleData</class>
 <widget class="QMainWindow" name="SampleData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>摄影机定标</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListView" name="sampleDataList">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>91</width>
      <height>341</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QTableView" name="dataInfo">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>451</width>
      <height>341</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>570</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>相机定标</string>
    </property>
    <addaction name="open_image"/>
    <addaction name="read_control_point"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="read_control_point">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/images/dingwei.png</normaloff>:/images/images/dingwei.png</iconset>
   </property>
   <property name="text">
    <string>读取控制点文件(&amp;O)</string>
   </property>
  </action>
  <action name="open_image">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/images/image.png</normaloff>:/images/images/image.png</iconset>
   </property>
   <property name="text">
    <string>打开影像 (&amp;O)</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
